src/server/server.js

```diff
@@ -1,40 +1,38 @@
-//=============================================================================
-// Nelderson's Online Core Server
-// Version: 0.2.1 - August 3rd, 2017
-//=============================================================================
 var express = require('express');
 var app = express();
 var server = require('http').Server(app);
 var io = require('socket.io')(server);
-var config = require('./configurations/config');
 var bodyParser = require('body-parser');
 var logger = require('morgan'); //For development
 var socketioJwt = require('socketio-jwt');
+
+var config = require('./configurations/mainConfig');
 var log = require('tracer').colorConsole(config.loggingConfig);
 
 app.use(logger('dev'));//For development
 app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.urlencoded({ extended: false }));
 
 server.listen(config.port);
-log.info('Server is on bruh and running on port: '+ config.port);
+log.info('Server is on bruh and running on port: ' + config.port);
 
 //----------------------------------
 // SET ROUTES FOR EXPRESS API HERE:
 //----------------------------------
 
+app.use(function (req, res, next) {
```

Comment:
Not requirt whit cors patch 👍

Reply:
Will update this PR at the weekend with latest changes so it's ready to merge